Bigtable/tests/Unit/Connection/GrpcTest.php

```diff
@@ -102,6 +106,47 @@ public function methodProvider()
         );
         $lro = $this->prophesize(OperationResponse::class)->reveal();
 
+        $tableId = 'my-table';
+        $table = $serializer->decodeMessage(
+            new Table(),
+            []
+        );
+
+        $columnFamily = $serializer->decodeMessage(
+            new ColumnFamily(),
+            []
+        );
+
+        $columnFamilyModificationCreate = $serializer->decodeMessage(
```

Comment:
Please add a todo comment here and for `columnFamilyModificationUpdate` to add GCRule.

Reply:
done